src/language/__tests__/parser-test.ts

```diff
@@ -120,6 +120,13 @@ describe('Parser', () => {
     });
   });
 
+  it('does not allow "true" as enum value', () => {
```

Comment:
Also please add tests for `false` and `null`.
You can keep them under a single test case just rename it to something like: does not allow "true", "false", and "null" as enum values.